lightning-network-book.tex

```diff
@@ -147,27 +147,26 @@ \section{Introduction to Electronic Cash}
 
 \subsection{Electronic coins as chains of digital signatures}
 \problem{How do you claim ownership and authenticity of a digital good or a piece of information?}
-\todo{Definately rethink the approach for this section. Also in the above chapter it might not have been wise to already introduce the double spending problem.}
-By the end of the day cash is about ownership.
-Therfore when creating an electronic cash system we have to be able to ascribe ownership of digital information. \todo{need to define if I want to talk about information or messages}
-Another goal is to be able to verify the authenticity of electronic cash.
-We already expect in our everyday live that we can do the same with a bank note.
+\todo{Definitely rethink the approach for this section. Also in the above chapter it might not have been wise to already introduce the double spending problem.}
+At the end of the day cash is about ownership.
+Therefore, when creating an electronic cash system we have to be able to ascribe ownership of digital information. \todo{need to define if I want to talk about information or messages}
+Another goal is to be able to verify the authenticity of electronic cash, as we do with bank notes in our everyday lives.
 As electronic cash is supposed to be a piece of information we can look in the field of computer science to see how ownership and authenticity of a piece of information can be claimed.
 The standard procedure would be to make use of \gls{digital signature}s.
 Interestingly in the digital world it becomes significantly harder to fake an electronic coin in comparison to a physical bank note.
-\footnote{Unless of course one does not handle ones private keys - which are used to create a digital signature -  properly.}
+\footnote{Unless of course one does not handle one's private keys - which are used to create a digital signature - properly.}
 
 
 \problem{How do digital signatures work?}
 A well known method to create digital signatures is by using an \gls{asymmetric cryptographic system}.
-Despite the fact that modern cryptography involves quite some advanced mathmatical theory and computer science I want to encourage you to stick with me in this section.
-Here we will just use this powerfull tool as a blackbox and describe its properties which should be easy to understand even for people who have always had trouble with mathematics.
-In the appendix\todo{put in label} we give some mathematical and technical background about elliptic curve cryptography and about how and why it actually works to provide secure digital signatures.
+Despite the fact that modern cryptography involves quite advanced mathmatical theory and computer science, I want to encourage you to stick with me in this section.
+Here we will consider this powerful tool as a blackbox and describe its properties, which should be easy to understand even for people who have trouble with mathematics.
```

Comment:
I will merge this. But just to clarify. Is it grammatically wrong to say "I will only make use of..." instead of I will consider to..." I kind of wanted to emphasize that we will just apply it.

Reply:
The word "just" is kind of ambiguous, so it's unclear to the reader how you meant to use it. It could be: "Here we will only use this powerful tool as a blackbox" or "Here we will simply use this powerful tool as a blackbox".

I interpreted it as the second meaning, which is why I rewrote it as "Here we will consider this powerful tool as a blackbox".